android/modules/media/src/java/ti/modules/titanium/media/MediaModule.java

```diff
@@ -1105,27 +1122,15 @@ public void openPhotoGallery(KrollDict options)
 			errorCallback = (KrollFunction) options.get(TiC.EVENT_ERROR);
 		}
 
-		final KrollFunction fSuccessCallback = successCallback;
-		final KrollFunction fCancelCallback = cancelCallback;
-		final KrollFunction fErrorCallback = errorCallback;
+		fSuccessCallback = successCallback;
+		fCancelCallback = cancelCallback;
+		fErrorCallback = errorCallback;
 
 		Log.d(TAG, "openPhotoGallery called", Log.DEBUG_MODE);
 
 		Activity activity = TiApplication.getInstance().getCurrentActivity();
 		TiActivitySupport activitySupport = (TiActivitySupport) activity;
 
-		TiIntentWrapper galleryIntent = new TiIntentWrapper(new Intent());
-		galleryIntent.getIntent().setAction(Intent.ACTION_GET_CONTENT);
```

Comment:
@m1ga Under the hood `ACTION_GET_CONTENT` already invokes new Photo picker when it's available, otherwise it invokes system picker. I tested it out on 12.5.1.GA on Android 15 device and it invokes new Photo picker.

Reply:
Scroll down to `ACTION_GET_CONTENT behaviour change` on [this page](https://medium.com/androiddevelopers/permissionless-is-the-future-of-storage-on-android-3fbceeb3d70a) for more details.

Reply:
ah that is good too know! I thought the new Photo Picker is _only_ used when you use PickVisualMediaRequest as they don't mention the ACTION_GET_CONTENT in their page.

Then I guess we don't need to do anything here :-) I'll put this into draft so we don't need to do anything here but still have to code